examples/decoding/plot_simulated_data.py

```diff
@@ -41,9 +41,9 @@ def create_simulation_data(snr=0, n_samples=2 * 100, size=12, random_state=1):
     w[-roi_size:, -roi_size:, 0:roi_size] = 0.5
```

Comment:
I can run this example in master with python 3 so I don't think this change is absolutely needed.

Using float as numpy array indices is deprecated indeed as the following snippet shows so feel free to open a separate PR with this change

```python

import warnings

import numpy as np

warnings.simplefilter('always', DeprecationWarning)

arr = np.arange(10)

print(arr[2.3:5.9])
```